src/store/item-store.js

```diff
@@ -1,26 +1,23 @@
 import { Store } from 'consus-core/flux';
-
+import { searchItem } from '../lib/api-client.js';
 let item = null;
 
 class ItemStore extends Store {
     getItem() {
-        if (item === null) {
-            return item;
-        }
-        return {
-            id: item.id,
-            status: item.status
-        };
+        return item;
+    }
+    getAllItems(){
+        return item;
+    }
+    searchItemByAddress(address){
```

Comment:
This is what you're doing:

![image](https://cloud.githubusercontent.com/assets/5055041/20815556/fb566ab0-b7e3-11e6-9436-7a44f7770f6f.png)